### ISSUE TITLE
Relative paths gets out of context

### ISSUE THREAD

**Issue by odadoda** [wtfil/image-set-polyfill#6]
I got relative paths to fonts and some svg's in the css. When the polyfill kicks in, it copies the css and places it inline in the document, where all the relative paths is out of the original context and can't find the resources.

**Comment by wtfil** [comment]
@odadoda 
Hi! This is sad. At this case simple regex is not enough, will fix that soon.

**Comment by odadoda** [comment]
I guess one solution would be to prepend the path to the css file into all "url(...)" values.

**Comment by wtfil** [comment]
yeah, it could work, but ideally it should not touch the @font-face and svg at all:)
